modules/commons/src/main/resources/schemas/neurosciencegraph/commons/selectors/v0.2.0.json

```diff
@@ -0,0 +1,176 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.2.0",
+    {
+      "@base": "{{base}}/schemas/neurosciencegraph/commons/selectors/v0.2.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "shapes": [
+    {
+      "@id": "SelectorShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "property": [
+        {
+          "path": "rdf:type",
+          "name": "Selector type",
+          "description": "The class of the Selector.",
+          "minCount": 1,
+          "maxCount": 1
+        }
+      ]
+    },
+    {
+      "@id": "TextPositionSelectorShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "class": "oa:TextPositionSelector",
+      "and": [
+        {
+          "node": "SelectorShape"
+        },
+        {
+          "property": [
+            {
+              "path": "oa:start",
+              "name": "Start offset",
+              "description": "The starting position of the segment of text. The first character in the full text is character position 0, and the character is included within the segment.",
+              "datatype": "xsd:integer",
+              "minInclusive": 0,
+              "lessThan": "oa:end",
+              "minCount": 1,
+              "maxCount": 1
+            },
+            {
+              "path": "oa:end",
+              "name": "End offset",
+              "description": "The end position of the segment of text. The character is not included within the segment.",
+              "datatype": "xsd:integer",
+              "minInclusive": 0,
```

Comment:
Maybe put minInclusive: 1 since the value can never be 0 (because the value is not included in the segment and the oa:start property needs to be less than the oa:end property)

Reply:
I agree. I have just done it because this is how it is described in https://www.w3.org/TR/annotation-vocab/#end.